docker-hub/vulnerability-scanning.md

```diff
@@ -1,124 +1,174 @@
 ---
-description: Hub Vulnerability Scanning
-keywords: scanning, vulnerabilities, Hub, Snyk
-title: Hub Vulnerability Scanning
+description: Hub Basic vulnerability scanning
+keywords: scanning, vulnerabilities, Hub, basic
+title: Basic vulnerability scanning
 ---
 
-{% include upgrade-cta.html
-  body="Docker Hub Vulnerability Scanning is available for users subscribed to Docker Pro, Team, or a Business subscription. Upgrade now to get automatic vulnerability scans for your images."
-  header-text="This feature requires a paid Docker subscription"
-  target-url="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade_scan"
-%}
+> **Note**
+>
+> Hub Vulnerability Scanning requires a Docker Pro, Team, or Business
+> subscription.
 
-Docker Hub Vulnerability Scanning enables you to automatically scan Docker
-images for vulnerabilities using Snyk. This uses the same technology as the
-[docker scan](../engine/scan/index.md) command.
+Docker Hub vulnerability scanning lets you automatically scan Docker images for
+vulnerabilities.
 
-> When you enable Hub Vulnerability Scanning, you can also see whether your
-> images are affected by Log4Shell (CVE-2021-44228). For more information, see
-> [Scan images](#scan-images-on-docker-hub).
+When you push an image to a Docker Hub repository after turning on vulnerability
+scanning, Docker Hub automatically scans the image to identify vulnerabilities.
+Vulnerability Scanning lets you review the security state of your images and
+take actions to fix issues identified during the scan, resulting in more secure
+deployments.
 
-When you push an image to Docker Hub after enabling vulnerability scanning, Docker Hub automatically scans the image to identify vulnerabilities in your container images. Vulnerability Scanning allows developers and development teams to review the security state of the container images and take actions to fix issues identified during the scan, resulting in more secure deployments. The scan result includes the source of the vulnerability, such as OS packages and libraries, version in which it was introduced, and a recommended fixed version (if available) to remediate the vulnerabilities discovered.
+Scan results include:
 
-## Scan images on Docker Hub
+- The source of the vulnerability, such as OS packages and libraries
```

Comment:
```suggestion
- The source of the vulnerability, such as Operating System (OS) packages and libraries
```

It's hard to see in diff view, but looks like this is the first instance of this acronym, so unpack it